backend/src/data/migrations/20220830101405_add_task_notes_table.ts

```diff
@@ -0,0 +1,60 @@
+import { Knex } from 'knex';
+
+enum TableName {
+  TASKS = 'tasks',
+  TASK_NOTES = 'task_notes',
+  USERS = 'users',
+}
+
+enum ColumnName {
+  ID = 'id',
+  CREATED_AT = 'created_at',
+  UPDATED_AT = 'updated_at',
+  TASK_ID = 'task_id',
+  AUTHOR_ID = 'author_id',
+  NOTE = 'note',
+  STATUS = 'status',
+}
+
+const DELETE_STRATEGY = 'CASCADE';
+
+const STATUS_VALUES = ['Uncompleted', 'Pending', 'Rejected', 'Completed'];
+
+const STATUS_UNCOMPLETED = 'Uncompleted';
```

Comment:
avoid duplications. create an enum instead. use `Object.values(enum)` to create an array of values
all values should be in lowercase